packages/experiments/src/components/Microfeedback/Microfeedback.state.ts

```diff
@@ -0,0 +1,18 @@
+import { IMicrofeedbackProps, IMicrofeedbackViewProps } from './Microfeedback.types';
+import { BaseState } from '../../utilities/BaseState';
+
+export interface IMicrofeedbackState extends IMicrofeedbackViewProps {
+  vote: number; // -1, 0, 1 based on dislike, no vote, like
```

Comment:
You might consider using string union types here which are a bit more descriptive than int enums i.e.

```ts
vote: 'dislike' | 'no_vote' | 'like';
```

Reply:
Fixed